test/router_test.exs

```diff
@@ -197,4 +197,56 @@ defmodule Kitto.RouterTest do
     assert conn.status == 401
     Application.delete_env :kitto, :auth_token
   end
+
+  test "POST /dashboards/:id reloads single dashboard" do
+    dashboard = "sample"
+    body = %{command: "reload"}
+    conn = conn(:post, "dashboards/#{dashboard}", Poison.encode!(body))
+
+    mock = fn (topic, data) ->
+      stringified_body = for {key, val} <- data, into: %{}, do: {String.to_atom(key), val}
+      if topic == "_kitto" && stringified_body == Map.put(body, :dashboard, dashboard) do
+        send self, :ok
+      end
+    end
+
+    with_mock Kitto.Notifier, [broadcast!: mock] do
+      Kitto.Router.call(conn, @opts)
+      assert_receive :ok
+    end
+  end
+
+  test "POST /dashboards reloads all dashboards" do
+    body = %{command: "reload"}
+    conn = conn(:post, "dashboards", Poison.encode!(body))
+
+    mock = fn (topic, data) ->
+      stringified_body = for {key, val} <- data, into: %{}, do: {String.to_atom(key), val}
```

Comment:
I'm guilty :blush: 

I suggest changing L#160 as well, `atomized_body` sounds better imho, we're just transforming the keys to atoms there.